drivers/storage/portworx/component/disruption_budget.go

```diff
@@ -151,9 +151,13 @@ func (c *disruptionBudget) createPortworxPodDisruptionBudget(
 				},
 			},
 		}
-		return k8sutil.CreateOrUpdatePodDisruptionBudget(c.k8sClient, pdb, ownerRef)
+
+		err = k8sutil.CreateOrUpdatePodDisruptionBudget(c.k8sClient, pdb, ownerRef)
+		if err == nil {
+			logrus.Infof("Value for px-storage pod disruption budget is set to %d ", pdb.Spec.MinAvailable.IntValue())
```

Comment:
This will print every time even if the value is not changing. Let's move this print to https://github.com/libopenstorage/operator/blob/master/pkg/util/k8s/k8s.go#L1675 and print it only when the existing and new min_available values don't match.